Index: src/main/java/UI/MederAbdymusaev/Utils.java
===================================================================
diff --git a/src/main/java/UI/MederAbdymusaev/Utils.java b/src/main/java/UI/MederAbdymusaev/Utils.java
deleted file mode 100644
--- a/src/main/java/UI/MederAbdymusaev/Utils.java	(revision 129d2d57fc0bfdea5214d78f423aea9b9530542a)
+++ /dev/null	(revision 129d2d57fc0bfdea5214d78f423aea9b9530542a)
@@ -1,4 +0,0 @@
-package UI.MederAbdymusaev;
-
-public class Utils {
-}
